src/test/java/org/alfresco/rest/api/search/SerializerTestHelper.java

```diff
@@ -127,4 +127,16 @@ public void writeContents(JsonGenerator generator, ObjectMapper objectMapper)
         });
         return out.toString();
     }
+    public <T> Object extractFromJson(String json, Class<?> clasz) throws IOException
```

Comment:
Why has this method been extracted? As far as I can tell the only place this method is called is the method below.

Reply:
Were you planning to refactor extractFromJson(String json) and use your new method? In that case you'll reduce code duplication and it would make sense.